tests/data/preview/long_strings.py

```diff
@@ -512,23 +568,31 @@ def foo():
 )
 
 func_with_bad_comma(
-    "This is a really long string argument to a function that has a trailing comma"
-    " which should NOT be there.",
+    (
+        "This is a really long string argument to a function that has a trailing comma"
+        " which should NOT be there."
+    ),
 )
 
 func_with_bad_comma(
-    "This is a really long string argument to a function that has a trailing comma"
-    " which should NOT be there.",  # comment after comma
+    (  # comment after comma
+        "This is a really long string argument to a function that has a trailing comma"
+        " which should NOT be there."
+    ),
 )
 
 func_with_bad_comma(
-    "This is a really long string argument to a function that has a trailing comma"
-    " which should NOT be there.",
+    (
+        "This is a really long string argument to a function that has a trailing comma"
+        " which should NOT be there."
+    ),
 )
 
 func_with_bad_comma(
-    "This is a really long string argument to a function that has a trailing comma"
-    " which should NOT be there.",  # comment after comma
+    (  # comment after comma
```

Comment:
hm, it seems like this comment gets moved to the beginning? That's unfortunate.

Reply:
yeah, i'm actually not sure where the comment should go. there is another existing example:

https://github.com/psf/black/blob/1b6de7b0a33b568f71ff86e0e5fef6d4c479c2b7/tests/data/preview/long_strings.py#L91

https://github.com/psf/black/blob/1b6de7b0a33b568f71ff86e0e5fef6d4c479c2b7/tests/data/preview/long_strings.py#L447-L450

that throws the comment to the opening paren, which is the same behavior here.

looks like it was introduced in https://github.com/psf/black/pull/1132, but it's unclear whether this behavior is intentionally designed or not.

thoughts?

Reply:
I see, thanks. Maybe we can fix this, but seems like the issue is independent from this PR.